### DISCUSSION TITLE
Custom Vercel Deployment Link Redirects to Login Screen Instead of Showing the Project

### DISCUSSION THREAD

**Discussion by janheinvanasseldonk** [vercel/community#6734] [Help] [answered]
### Summary

Hello,

My custom Vercel link (x.vercel.app) suddenly stopped working and now redirects users to the Vercel login page (https://vercel.com/login?next=...). This issue started after some recent operations on branches and environment settings.

Here are the steps I've already taken to troubleshoot:

- Verified the deployment URL and checked for typos.
- Confirmed the deployment exists and is visible on my Vercel dashboard.
- Reviewed the deployment logs, but didn't find any clear errors that could explain this behavior.
- Ensured that there are no access restrictions that might be causing this.

Despite these efforts, the problem persists. Could you please provide some guidance on what might be causing this issue or any additional steps I could take to resolve it?

Thank you for your assistance!

### Example

Removed

### Steps to Reproduce

Removed

**Comment by jay-batavia** [comment]
Our team is facing a similar issue. Our _custom domain for a specific branch_ now redirects to the Vercel login page instead of navigating to the project homepage. This only started happening within the last week, and we have made no changes to how the domains are configured within the project.

The "Assigning Custom Domains" section on the preview deployment page says it has been assigned correctly. I have removed and reconfigured the domain in Project Settings a few times now, and nothing seems to be working. FWIW our custom domain for production deployments seems to be working fine.

**Reply by jay-batavia** [reply, marked as answer]
^ This has been resolved for us. I was able to get in touch with support and they informed us of [Deployment Protection](https://vercel.com/docs/security/deployment-protection). 

It's still strange that it _was_ working and then changed without notice. Anyway, hopefully that helps you.

**Reply by janheinvanasseldonk** [reply]
Glad to hear! Same here, somehow Vercel Authentication was enabled (not by me), which made it seem as if the links didn't work any longer.
